charts/bindplane/templates/bindplane-nats.yaml

```diff
@@ -0,0 +1,356 @@
+{{- if eq .Values.eventbus.type "nats" }}
```

Comment:
This statefulset is based on the existing template bindplane.yaml (and bindplane-jobs.yaml).